packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/createFolderView.ts

```diff
@@ -0,0 +1,19 @@
+import { DEFAULT_ACTION_VIEW_DISPLAY_TEXT } from './shared';
+import { DefaultCreateFolderViewDisplayText } from '../../types';
+
+export const DEFAULT_CREATE_FOLDER_VIEW_DISPLAY_TEXT: DefaultCreateFolderViewDisplayText =
+  {
+    ...DEFAULT_ACTION_VIEW_DISPLAY_TEXT,
+    title: 'Create folder',
+    actionStartLabel: 'Create folder',
+    getValidationMessage: (_error) => {
+      // @TODO: maybe add logic to check which character caused the failure
+      return `Folder name must be at least one character and cannot contain a "/", ".", or ".."`;
```

Comment:
Think we can drop the piece regarding a single character since we disable the start button for an empty string, does that make sense?